xinference/model/llm/llm_family.json

```diff
@@ -461,6 +461,66 @@
       }
     ]
   },
+  {
+    "version": 1,
+    "context_length": 128000,
+    "model_name": "phi-3-mini-128k-instruct",
+    "model_lang": [
+      "en"
+    ],
+    "model_ability": [
+      "generate"
```

Comment:
Actually, this model is a chat model, so you need to add chat ability, hence, some fields might be absent, like https://github.com/qinxuye/inference/blob/78897c51ca7c0a4751a870dd0b8b6813a7eafd58/xinference/model/llm/llm_family.json#L1346-L1361

Reply:
Thank you. I will try to add chat ability.

Reply:
You can refer to #1384 , this is another contribution from the community.